sirepo/job_api.py

```diff
@@ -49,7 +49,13 @@ def api_admJobs(self):
     def api_analysisJob(self):
         return self.request()
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec(
+        "require_user",
+        sid="SimId",
+        model="Model",
+        frame="Frame",
+        suffix="String optional",
```

Comment:
I think names like Model and Frame are too generic. Model l means two things: the actual data and the model name, which is confusing. In this case, I think this is a ComputeModel, but I could be wrong. @e-carlin what say you?

Frame is a number. It really is an index, so maybe DataFileIndex.

String should be constrained. At is a FileSuffix, and it should be limited to 30 characters or something like that.

Reply:
Wouldn't that be an AnalysisModel? The user may be clicking the downloadDataFile on the fooAnimation panel  and underneath the computeModel may be something else. The model that comes in I thought was the AnalysisModel.

Reply:
24fd189959a5d6096a9482e74f1ec55c9faeb6de